### ISSUE TITLE
Configure DataONE additional account

### ISSUE THREAD

**Issue by craig-willis** [whole-tale/dashboard#573]
As noted in https://github.com/whole-tale/dashboard/pull/563, "DataONE is present, but not yet functional".  

* [x] Connect additional accounts page displays DataONE providers (e.g., DataONE staging) (done)
* [x] User selects "Connect" button and is redirected through ORCID flow. As a result, DataONE token is stored in `otherTokens`
* [x] User selects "Disconnect" and token/configured account information is removed.
* [ ] Callback to Girder with the DataONE JWT when a `preauthorized` token is detected.

**Comment by craig-willis** [comment]
Per 12/2 dev call, we're likely not handling the DataONE flow correctly in the UI. See https://github.com/whole-tale/girder_wholetale/pull/364#issue-331121398 DataOne section

**Comment by bodom0015** [comment]
As written, the first 3 checkboxes above absolutely were fulfilled by #563. I've added a 4th that highlights the functionality that still needed to be added. The missing piece was apparently, as you had mentioned, that the callback to Girder that wasn't in the test case when I started reviewing [https://github.com/whole-tale/girder_wholetale/pull/364](that PR).

In any case, I've created a [PR](https://github.com/whole-tale/dashboard/pull/575) to add the missing logic for Connecting a DataONE account via the Dashboard
